dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/services/DatabaseStringDataStore.kt

```diff
@@ -262,23 +261,11 @@ class DatabaseStringDataStore(
      * @param correlationId the correlationId ot the current user process
      */
     @Transactional(propagation = Propagation.NEVER)
-    fun deleteDataItemWithoutTransactionAndSendMessage(
+    fun deleteDataItemWithoutTransaction(
```

Comment:
Because of the annotation, no transaction will be created which results in this action being commited instantly, which (if i remember correctly) was important